emoji_playground.py

```diff
@@ -134,18 +138,25 @@ async def _fetch_and_display():
     original_image.src = window.URL.createObjectURL(original_image_file)
     document.getElementById("original_image").appendChild(original_image)
 
-select_emoji_and_display = create_proxy( _select_emoji_and_display)
-document.getElementById("emoji-selector").addEventListener("change",  select_emoji_and_display)
-
-select_filter_and_display = create_proxy( _select_filter_and_display)
-document.getElementById("filter-selector").addEventListener("change",  select_filter_and_display)
-
 def remove_all_children(parent_id: str):
     parent = document.getElementById(parent_id)
 
     while parent.firstChild is not None:
         parent.removeChild(parent.firstChild)
 
-await _fetch_and_display()
+if "pyodide" in sys.modules:
+
+    select_emoji_and_display = create_proxy( _select_emoji_and_display)
+    document.getElementById("emoji-selector").addEventListener("change",  select_emoji_and_display)
+
+    select_filter_and_display = create_proxy( _select_filter_and_display)
+    document.getElementById("filter-selector").addEventListener("change",  select_filter_and_display)
+
+    async def _call_fetch_and_display(): # no idea if this is the right way of doing this
+        await _fetch_and_display()
+
+    _call_fetch_and_display()
```

Comment:
This gives now the warning from the webpage https://kolibril13.github.io/pyscript-emoji-skimage/pr-preview/pr-6/ 
`:158: RuntimeWarning: coroutine '_call_fetch_and_display' was never awaited RuntimeWarning: Enable tracemalloc to get the object allocation traceback`

but when I keep the `await _fetch_and_display()` function like it was before, I get an error when I type `from emoji_playground import *` in the jupyter notebook:
```
  File ~/projects/pyscript-emoji-skimage/emoji_playground.py:155
    await _fetch_and_display()
    ^
SyntaxError: 'await' outside function
```